### ISSUE TITLE
Scuttle ship button

### ISSUE THREAD

**Issue by mitm001** [SoftbearStudios/mk48#199]
A new button that allows you to scuttle your ship is now called for since ships no longer can use terrain to suicide.

The lack of control over spawning causes ships to use suicide/scuttle due to poor spawning area. Sometimes you just don't want to spawn on the other side of the map. All penalties that apply now would still be retained for scuttle.

**Comment by finnbear** [comment]
I agree! The trick is to add it without the potential for accidents (maybe needs a confirmation) and without cluttering the GUI too much (maybe an opt-in setting to show it)
